Source/NSTextList.m

```diff
@@ -40,6 +40,7 @@ - (id) initWithMarkerFormat: (NSString *)format
 {
   ASSIGN(_markerFormat, format); 
   _listOptions = mask;
+  _startingItemNumber = 1;
```

Comment:
We are not serializing or deserializing this. I think that's bad.

`-isEqual:` has also not been updated.

We should also add a TODO into `-markerForItemNumber:` to take `_startingItemNumber:` into account.